src/components/TopBar/HamburgerMenu.jsx

```diff
@@ -37,6 +38,17 @@ const HamburgerMenu = createMenu({
         </MenuItem>,
       );
 
+      if (isUserAuthenticated) {
```

Comment:
Most of the places we are calling this should be changed to `isUserAuthenticatedWithGoogle` or `isUserAuthenticatedWithGithub` now that both are possible.